src/pages/DetailPages/index.js

```diff
@@ -34,7 +34,7 @@ function DetailPages(props) {
       for (const key in sprites) {
         const altText = formatCarouselCaption(key);
         const src = sprites[key];
-        if (altText && src) {
+        if (altText && src && key !== "versions" && key !== "other" ) {
```

Comment:
so the response change the `key` huh, well this might do the fix, but still shiny image is missing.